add KO_DEFAULTBASEIMAGE usage to docs (#895)
ko-build/ko: docs/configuration.md

defaultBaseImage: registry.example.com/base/image
```

> As an alternative way, you can use the `KO_DEFAULTBASEIMAGE` special environment variable to set the default base image to be used in `ko` builds as well:

Reviewer: ```suggestion
You can also use the `KO_DEFAULTBASEIMAGE` environment variable to set the default base image, which overrides the YAML configuration:
```
